firmware/integration/rusefi_config.txt

```diff
@@ -1820,8 +1820,9 @@ uint8_t autoscale knockFuelTrim;Fuel trim when knock, max 30%;"%", 1, 0, 0, 30,
 
 	int8_t[8] airmassToTimingBins;;"mg", 1, 0, -100, 100, 0
 	int8_t[8] airmassToTimingValues;;"deg", 1, 0, -30, 30, 0
+	uint8_t autoscale idleReturnTargetRampDuration;idle return target ramp duration;"seconds", 0.1, 0, 0, 15, 1
```

Comment:
is zero a legit value here?

Reply:
@ElDominio may be able to help with the definition. Less than 1 second of ramp doesn't seem like a valid value to me. For now i've updated it to 0.1 and added the validation.